### PR TITLE
#No issue-Created Cards and Footer With Mock Data- HASEEB_KHALIL

### PR DESCRIPTION
**Pull request by Haseeb3927**
This includes All Mock Data which is being displayed in cards and footer. It should be replaced with real data once database is ready.

## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.
Just created cards and footer using Mock Data.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [ * ] My code follows the style guidelines of this project
- [ * ] I have carefully reviewed my own code
- [ * ] I have commented my code
- [ * ] I have updated any documentation

### CODE REVIEW

File: client/src/pages/mainPage/components/footer/Footer.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
+footer {
+  background: var(--color-primary);
```
Comment (Luke-Rogerson) [inline client/src/pages/mainPage/components/footer/Footer.css:2]: Nice use of CSS vars!

File: client/src/pages/mainPage/components/footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import "./Footer.css"
+import{BsGithub} from "react-icons/bs"
+import { BsLinkedin } from "react-icons/bs";
+
+export default function Footer() {
+  return (
+		<footer>
```
Comment (Luke-Rogerson) [inline client/src/pages/mainPage/components/footer/Footer.js:8]: Perhaps format this? In VSCode I think you can run `Shift + Alt + F`

File: client/src/pages/mainPage/components/home/Home.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
+/* Hero Styles */
+.booster__hero{
+	background-image:url("../energiser-Imgs/Booster-hero-img.png");
+	/* background-position: center;
```
Comment (Luke-Rogerson) [inline client/src/pages/mainPage/components/home/Home.css:4]: Try and not merge commented out code to main branch

File: client/src/pages/mainPage/components/home/Home.js [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+		Image: IMG,
+		Description:
+			"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+		link: "https://sample.com",
+		id: "10",
+	},
+];
+
+export function Home() {
+	// const [energizers, setEnergizers] = useState();
```
Comment (Luke-Rogerson) [inline client/src/pages/mainPage/components/home/Home.js:95]: You could add `[]` as the default argument here so you don't need to check if energisers is null

**Comment by Luke-Rogerson** [comment]
Looking good. You will need to resolve the merge conflict in `server/db.js` first - https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line

File: client/src/pages/mainPage/components/footer/Footer.css (hunk 1 above)
Comment (Haseeb3927) [reply client/src/pages/mainPage/components/footer/Footer.css:2]: Thank you

File: client/src/pages/mainPage/components/footer/Footer.js (hunk 2 above)
Comment (Haseeb3927) [reply client/src/pages/mainPage/components/footer/Footer.js:8]: Sure

File: client/src/pages/mainPage/components/home/Home.css (hunk 3 above)
Comment (Haseeb3927) [reply client/src/pages/mainPage/components/home/Home.css:4]: I'll fix it now

File: client/src/pages/mainPage/components/home/Home.js (hunk 4 above)
Comment (Haseeb3927) [reply client/src/pages/mainPage/components/home/Home.js:95]: Thank you.
I'll be deleting this as soon as we're able to fetch from the database.

**Review by Tandy-Nicole — APPROVED** [review]
Great job @Haseeb3927

Well structured and keep up the good work.

**Pull request merged**